inference-engine/tests/functional/plugin/shared/src/single_layer_tests/activation.cpp

```diff
@@ -78,6 +78,17 @@ InferenceEngine::Blob::Ptr ActivationLayerTest::GenerateInput(const InferenceEng
             data_range = 2;
             break;
         }
+        case ngraph::helpers::ActivationTypes::Acosh: {
+            data_start_from = 1;
+            data_range = 20;
+            break;
+        }
+        case ngraph::helpers::ActivationTypes::Atanh: {
+            // TODO: Atanh domain is (-1, 1), but here expresses [-1, 1] and fails test, so uses 0 instead
```

Comment:
It would be nice to test more than one value, but I agree with leaving this as a `TODO` for now.